lua/entities/gmod_wire_expression2/core/extloader.lua

```diff
@@ -81,6 +81,8 @@ end
 local function e2_include_pass2(name, luaname, contents)
 	local preprocessedSource = e2_extpp_pass2(contents)
 
+	E2Lib.currentextension = string.gsub(name,"%.lua","")
```

Comment:
string.StripExtension

Reply:
good idea

Reply:
nevermind, @Divran checked the source and StripExtension is slightly less efficient...
https://github.com/Facepunch/garrysmod/blob/master/garrysmod/lua/includes/extensions/string.lua#L125-L129